Documentation/Contributing/CodingGuidelines.md

```diff
@@ -171,7 +171,7 @@ Always declare an access modifier for all fields, properties and methods.
 protected int myVariable = 0;
 
 // property should have protected setter
-public int MyVariable { get { return myVariable; } }
+public int MyVariable { get => myVariable; }
```

Comment:
For this one could we even go one step further, or would that muddle the guideline?

```suggestion
public int MyVariable => myVariable;
```